contracts/event/MultipleResultsEvent.sol

```diff
@@ -562,4 +519,139 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
         uint increment = _thresholdPercentIncrease.mul(currentThreshold).div(100);
         return currentThreshold.add(increment);
     }
+
+    /// @notice Calculates the tokens returned for a non-Invalid final result.
+    /// @param better Address to calculate winnings for.
+    /// @return Amount of tokens that will be returned.
+    function calculateNormalWinnings(
+        address better)
+        private
+        view
+        returns (uint)
+    {
+        // Calculate user's winning bets
+        uint myWinningBets;
+        uint myWinningVotes;
+        for (uint8 i = 0; i <= _currentRound; i++) {
+            uint bets = _eventRounds[i].balances[_currentResultIndex].bets[better];
+            if (i == 0) {
+                myWinningBets = myWinningBets.add(bets);
```

Comment:
You can remove this, because you can do it outside of loop

Reply:
logic deprecated with new logic